lib/standalone.mjs

```diff
@@ -118,6 +102,19 @@ export default async function standalone({
 		validate,
 	});
 
+	/** @type {Formatter} */
```

Comment:
This should have been just before `const formatterFunction = await getFormatter(stylelint);`.